src/components/tx/TxTextInput.js

```diff
@@ -9,7 +9,34 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 class TxTextInput extends React.Component {
-  render() {
```

Comment:
These changes are also part of the other PR

Reply:
PR merged with master. Waiting Review.🙂